subprojects/core/src/main/java/org/gradle/api/internal/AbstractTask.java

```diff
@@ -94,19 +94,24 @@
 
     private final TaskIdentity<?> identity;
 
-    private final ProjectInternal project;
+    // Weird name to work around https://issues.apache.org/jira/browse/GROOVY-8732
+    private final ProjectInternal _project;
```

Comment:
We can reap the benefits of these renames when everyone uses at least Gradle 4.10 to compile, right? So we could remove `FileOperations` from `ProjectInternal` in Gradle 6.0?

Reply:
`ProjectInternal` does not implement `FileOperations`, only `DefaultProject` does. It would need to remain so as long as we want to support plugins compiled against Gradle 4.9 or earlier.